### PR TITLE
[DOC] 0.7.2 release note

### PR DESCRIPTION
**Pull request by minahlee**
### What is this PR for?
Release note for 0.7.2

### What type of PR is it?
Documentation

### Screenshots (if appropriate)
Download:
screencapture-localhost-4000-download-html-1497289191863

Release note:
screencapture-localhost-4000-releases-zeppelin-release-0-7-2-html-1497289224640

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: releases/zeppelin-release-0.7.2.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+## Apache Zeppelin Release 0.7.2
+
+The Apache Zeppelin community is pleased to announce the availability of the 0.7.2 release.
+
+The community put significant effort into improving Apache Zeppelin since the last release.
```
Comment (jongyoul) [inline releases/zeppelin-release-0.7.2.md]: The community puts significant efforts ?

Comment (minahlee) [inline releases/zeppelin-release-0.7.2.md]: I used it as past tense ;)

Comment (jongyoul) [inline releases/zeppelin-release-0.7.2.md]: Aha~~ ㅠㅠ

File: releases/zeppelin-release-0.7.2.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<br />
+You can visit [issue tracker](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12340231&projectId=12316221) for full list of issues have been resolved.
+
+
+<br />
+### Contributors
+
+This release would not be possible without the following community members' contributions:
+
+Jeff Zhang, 1ambda,  Lee moon soo, Prabhjyot Singh, soralee, AhyoungRyu, Khalid Huseynov, Nelson Costa, Benoy Antony, Christian Fries, CloverHearts, Vipin Rathor, LeiWang, Mina Lee, Jongyoul Lee, sravan, Frank Rosner, Paul Brenner, Elek Márton, Renjith Kamath, Sohaib Iftikhar, William Markito Oliveira, andrea, sergeymazin
```
Comment (minahlee) [inline releases/zeppelin-release-0.7.2.md]: I use the name that user set in their git config, I cannot know full name of the author if there is no information in the git log :(

**Pull request closed without merging**